sfputil/main.py

```diff
@@ -408,6 +429,19 @@ def logical_port_name_to_physical_port_list(port_name):
     else:
         return [int(port_name)]
 
+def logical_port_to_physical_port_index(port_name):
```

Comment:
is this function available in some other library? can we reuse?

Reply:
i couldn't find one in sonic-utilities